src/pages/benefits/viewAllBenefit/ApplicationDetails.tsx

```
@@ -35,83 +36,46 @@ const ApplicationDetails: React.FC = () => {
-              ? "Yes"
-              : "No",
+            name: `${applicantDetails.firstName} ${applicantDetails.middleName} ${applicantDetails.lastName}`,
+            applicationStatus: applicationData.status,
+            studentId: applicantDetails.studentId,
+            disabilityStatus: applicantDetails.disabilityType ? "Yes" : "No",
           },
         ]);
 
-        setDocuments(mockResponse.document);
+        // Map documents from applicationFiles
+        const documents = applicationData.applicationFiles.map((file: any) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)